backend/de.metas.business/src/main/java/de/metas/banking/api/BankAccountService.java

```diff
@@ -123,4 +123,14 @@ public Optional<BankAccountId> getBankAccountIdByIBAN(@NonNull final String iban
 	{
 		return bankAccountDAO.getBankAccountIdByIBAN(iban);
 	}
+
+	@NonNull
+	public String getBankNameIfAny(@NonNull final BankAccountId bankAccountId)
+	{
+		return Optional.ofNullable(bankAccountDAO.getById(bankAccountId))
+				.map(BankAccount::getBankId)
+				.flatMap(bankId -> Optional.ofNullable(bankRepo.getById(bankId)))
```

Comment:
isn't this 
.map(bankId -> bankRepo.getById(bankId))

imho u don't need flatMap just to wrap an optional for it

pls
1. either use Optional correctly
2. either go without optionals at all
3. the return value shall be the actual bank name or if Optional.empty()
4. so in that case `getBankName` would be a better name (drop the `Any` sufix)